build/templates/errors.py.mako

```diff
@@ -42,12 +44,12 @@ class Error(_ErrorBase):
         super(Error, self).__init__(error_code, error_description)
 
 
-class Warning(_ErrorBase):
+class ${module_name_class}Warning(Warning):
     '''A warning originating from the ${driver_name} driver'''
 
-    def __init__(self, error_code, error_description):
-        assert (_is_warning(error_code)), "Should not raise Warning if error_code is not positive."
-        super(Warning, self).__init__(error_code, error_description)
+    def __init__(self, warning_code, warning_description):
```

Comment:
Alternatively, we can remove warning_ and error_ prefixes from all. More consistent, and we are within the context of Warning or Error class anyway so it's not ambiguous.

I think it's cleaner.

Reply:
Remove all 'error_' and 'warning_' prefixes